src/Angor/Shared/Services/NetworkService.cs

```diff
@@ -104,6 +106,7 @@ public async Task CheckServices(bool force = false)
                         {
                             _logger.LogError($"Failed to check indexer status url = {indexerUrl.Url}, StatusCode = {response.StatusCode}");
                         }
+                        OnStatusChanged?.Invoke();
```

Comment:
Interesting this means we don't need to wait for response it will update the page when it arrives